engine/runtime/src/test/scala/org/enso/interpreter/test/instrument/RuntimeComponentsTest.scala

```diff
@@ -216,13 +234,18 @@ class RuntimeComponentsTest
     )
     val responses =
       context.receiveAllUntil(
-        context.executionComplete(contextId),
-        timeout = 120
+        Seq(
+          context.executionComplete(contextId),
+          context.analyzeJobFinished,
+          context.analyzeJobFinished
```

Comment:
This test now checks that there are two `analyzeJobFinished` messages delivered, right? Why there are two such messages?